src/cmd/sql_cmd.h

```diff
@@ -42,6 +42,7 @@ DEFINE_string(spark_conf, "", "The config file of Spark job");
 DECLARE_string(zk_cluster);
 DECLARE_string(zk_root_path);
 DECLARE_int32(zk_session_timeout);
+DECLARE_uint32(get_table_status_interval);
```

Comment:
this gflags has no effect in sdk

Reply:
oh. yes. I should move it to the `sql_cmd_test.cc`